MediaGalleryUi/Test/Mftf/ActionGroup/AdminAssertMediaGalleryAssetDeletedActionGroup.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminAssertMediaGalleryAssetDeletedActionGroup">
+        <annotations>
+            <description>Verify image has been deleted from the Enhanced media gallery.</description>
+        </annotations>
+        <arguments>
+            <argument name="imageName" type="string" defaultValue=""/>
+        </arguments>
+
+        <see userInput='You have successfully removed the image "{{imageName}}"' stepKey="verifyDeleteImage"/>
```

Comment:
Let's add this verification to existing `AssertEnhancedMediaGalleryImageDeletedActionGroup` (it would be good to rename that actionGroup to `AssertAdminEnhancedMediaGalleryImageDeletedActionGroup`)